.changeset/drain-release-guard-and-ordering-tests.md

```
@@ -0,0 +1,52 @@
+---
+'@rundown-org/core': patch
+---
+
+# Refuse a thenable `releaseOnCommit`, and pin the ordering its correctness rests on
+
+Follow-up to the completion drain's transactional Run Release (#794, merged in
+#839), from a second review round over the same diff.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)